src/cogs/version.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from discord.ext import commands
+
+MAJOR_VERSION = 0
+MINOR_VERSION = 1
+PATCH_VERSION = 0
+RELEASE_CANDIDATE = 1
+
+
+class Version(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.logging = logging.getLogger(__name__)
```

Comment:
Can you add a log statement that just logs the current version to `info` for easier debug?

Reply:
Added as of [9e9a8df](https://github.com/fjebaker/e-bot/pull/50/commits/9e9a8dfc7e9ff9aab7ea4e8fe5d072c955d87952) - I also made the command text more descriptive :)